nix/sources.json

```diff
@@ -1,14 +1,26 @@
 {
     "agent-rs": {
```

Comment:
`agent-rs` had to be bumped since we bumped `nixpkgs` which bumped `pkgs.openssl`, which was too newer for the older `agent-rs`'s `rust-openssl` to handle.

(i.e. compiling the older `agent-rs` using newer `nixpkgs` fails on `rust-openssl`)